lib/create-organization-api.js

```diff
@@ -142,46 +141,64 @@ export default function createOrganizationApi ({ http }) {
    * .catch(console.error)
    */
   function createTeamMembership (teamId, data = {}) {
-    return http.post('teams/' + teamId + '/team_memberships', data, {headers: teamAlphaHeaders})
+    return http.post('teams/' + teamId + '/team_memberships', data)
       .then((response) => wrapTeamMembership(http, response.data, teamId), errorHandler)
   }
 
   /**
-   * Gets an Team Membership
+   * Gets an Team Membership from the team with given teamId
    * @memberof ContentfulOrganizationAPI
+   * @param {string} teamId - Team ID
    * @param {string} id - Team Membership ID
    * @return {Promise<TeamMembership>} Promise for an Team Membership
    * @example
    * const contentful = require('contentful-management')
    *
-   * client.getOrganizations()
+   * client.getOrganization('organizationId')
    * .then((organization) => organization.getTeamMembership('teamId', 'teamMembership_id'))
    * .then((teamMembership) => console.log(teamMembership))
    * .catch(console.error)
    */
   function getTeamMembership (teamId, teamMembershipId) {
-    return http.get('teams/' + teamId + '/team_memberships/' + teamMembershipId, {headers: teamAlphaHeaders})
+    return http.get('teams/' + teamId + '/team_memberships/' + teamMembershipId)
       .then(response => wrapTeamMembership(http, response.data, teamId), errorHandler)
   }
 
   /**
-   * Gets an Team Membership
+   * Get all Team Memberships for the team
    * @memberof ContentfulOrganizationAPI
    * @param {string} id - Team Membership ID
    * @return {Promise<TeamMembership[]>} Promise for a Team Membership Collection
    * @example
    * const contentful = require('contentful-management')
    *
-   * client.getOrganizations()
+   * client.getOrganization('organizationId')
    * .then((organization) => organization.getTeamMemberships('teamId'))
    * .then((teamMemberships) => console.log(teamMemberships))
    * .catch(console.error)
    */
   function getTeamMemberships (teamId, query = {}) {
-    return http.get('teams/' + teamId + '/team_memberships', {headers: teamAlphaHeaders, query})
+    return http.get('teams/' + teamId + '/team_memberships', {query})
       .then(response => wrapTeamMembershipCollection(http, response.data, teamId), errorHandler)
   }
 
+  /**
+   * Gets all Team Memberships of the organization
+   * @memberof ContentfulOrganizationAPI
+   * @return {Promise<TeamMembership[]>} Promise for a Team Membership Collection
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * client.getOrganization('organizationId')
+   * .then((organization) => organization.getOrganizationTeamMemberships())
+   * .then((teamMemberships) => console.log(teamMemberships))
+   * .catch(console.error)
+   */
+  function getOrganizationTeamMemberships (query = {}) {
```

Comment:
I personally think `getOrganizationTeamMemberships` is confusing alongside `getTeamMemberships`. 

I think I'd prefer one method, `getTeamMemberships(opts)` where opts is an object that may or may not contain team ID. If it contains team ID, it gets team memberships for that team. If it doesnt, it gets team memberships for the whole org.

This is a new pattern for the SDK, but we also dont have any other entities like this, and I think it's better than having two different methods.

I'm open to reasons to keep it this way, though, if you all feel strongly.

Reply:
ex: opts can include `{query: {blah}, teamId: <teamId}`

Reply:
I personally think they should be different because of the different wrap teamMembership methods. These are different use cases that I tried to explain with the docs and additional info in the doc-app. The endpoint `organization/orgid/team_memberships` only has [GET] implemented and there is no story to implement show, update, delete. To manage particular teamMemberships one should use `getTeamMembership(teamId, teamMembershipId)`.

Reply:
maybe naming can be different
`getAllOrganizationTeamMemberships()`
`getTeamMemberships(teamId)` (I think the arg and docs specifies that it is by team since it the name is not nice if getTeamMembershipsByTeam)
`getTeamMembership(teamId, teamMembershipId)`

Reply:
see my comment below, wrapping the same entity with a different method also seems like the wrong approach to me.